app/components/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,39 @@
+"use strict"
+import dom from '../../utils/dom'
+import {log} from '../../utils/logger'
+import ToggleCalendar from 'ToggleCalendars'
+
+import {renderCalendar,populateGoogleDates} from '../../utils/calendar-utils'
+
+
+
+/**
+ * Calendar Component
+ * @param  {array} calendarEvents - The events for calendar to display
+ * @return {VNode}
+ */
+export default ({state: {sources,calendarEvents,route}}) => {
+console.log(sources);
+  return (
+      <section>
+        <ToggleCalendar state={sources}></ToggleCalendar>
+        <div id="calendar" onload={renderCal(sources)} />
```

Comment:
ok thanks sir. I was in a furry when i used that call to onload and it slipped my mind that I had used it.